.github/workflows/github-actions-workflow.yaml

```diff
@@ -0,0 +1,83 @@
+name: drone-gae
+
+on:
+  push:
+    branches: [ "main" ]
+    tags:
+      - '*'
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  GO111MODULE: on
+  CGO_ENABLED: 0
+ 
+jobs:
+   build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      actions: write
+      checks: write
+      contents: write
+      deployments: write
+      id-token: write
+      issues: write
+      pull-requests: write
+      security-events: write
+      statuses: write
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Test
+      run: |
+        go mod download
+        go test -cover -vet all
+    
+    - name: Build
+      run: |
+        go build -a -ldflags "-X main.version=${{ inputs.github_ref_name }} -X main.rev=${{ inputs.github_sha }}"
```

Comment:
are these inputs are being interpolated at build time? looking at https://github.com/nytimes/drone-gae/actions/runs/4107773919/jobs/7087659487#step:5:2 , it seems they are evaluating to empty strings?

Reply:
Ah, thanks. It was from a reusable workflow where I was passing these as an input parameters.